src/test/test-memfd-util.c

```diff
@@ -11,21 +12,39 @@
 
 TEST(memfd_get_sealed) {
 #define TEST_TEXT "this is some random test text we are going to write to a memfd"
-        _cleanup_close_ int fd = -EBADF;
-
-        fd = memfd_new_full("test-memfd-get-sealed", MFD_ALLOW_SEALING);
+        _cleanup_close_ int fd = memfd_new_full("test-memfd-get-sealed", MFD_ALLOW_SEALING);
         if (fd < 0) {
-                assert_se(ERRNO_IS_NOT_SUPPORTED(fd));
-                return;
+                ASSERT_TRUE(ERRNO_IS_NOT_SUPPORTED(fd));
+                return (void) log_tests_skipped_errno(fd, "Failed to create new memfd");
         }
 
-        assert_se(write(fd, TEST_TEXT, strlen(TEST_TEXT)) == strlen(TEST_TEXT));
+        int r;
+        unsigned seals;
+        ASSERT_OK(memfd_get_seals(fd, &seals));
+        r = memfd_get_seals(fd, &seals);
+        log_info("r: %d\n", r);
+        log_info("seals: %u\n", seals);
```

Comment:
no need for these logs

Reply:
Note memfd_get_seals was made static in https://github.com/systemd/systemd/pull/36980/commits/dba777cf26889786018d434935fc3d4aa8e5bbb0 it may be wise to rebase to account for that.